api/v1alpha1/cnbi_condition_consts.go

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Question: why not leave all the CNBi-related types and consts within `customnbimage_types.go`?

Reply:
for clearity